src/ocm/components/clusterConfiguration/DiscoveryImageTypeDropdown.tsx

```diff
@@ -69,12 +69,6 @@ export const DiscoveryImageTypeDropdown = ({
       key="discovery-image-ipxe"
       id="discovery-image-ipxe"
       description={'Use when you have an iPXE server that has already been set up.'}
-      tooltip={
-        imageTypeIsDisabled ? (
```

Comment:
Can we rename this variable to something like `isIPXEDisabled` to make it more explicit?

Reply:
Yes. we can

Reply:
Yeah, of course. :sweat_smile: